lib/generators/manageiq/plugin/templates/README.md

```diff
@@ -1,6 +1,6 @@
 # <%= class_name %>
 
-[![Build Status](https://travis-ci.com/ManageIQ/<%= plugin_name %>.svg?branch=master)](https://travis-ci.com/ManageIQ/<%= plugin_name %>)
+[![CI](https://github.com/ManageIQ/<%= plugin_name %>/actions/workflows/ci.yaml/badge.svg)](https://github.com/ManageIQ/<%= plugin_name %>/actions/workflows/ci.yaml)
 [![Maintainability](https://api.codeclimate.com/v1/badges/<badge_token>/maintainability)](https://codeclimate.com/github/ManageIQ/<%= plugin_name %>/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/<badge_token>/test_coverage)](https://codeclimate.com/github/ManageIQ/<%= plugin_name %>/test_coverage)
```

Comment:
Not for this PR, but I found a way to use just the repo name so we don't need the badge tokens., which I was considering blasting out after all of the actions stuff is done.  See an example here: https://github.com/ManageIQ/log_decorator/blob/f5d9a443b90e883233733bf9df1e8058cc9d3266/README.md?plain=1#L5-L6